### DISCUSSION TITLE
srsRAN gNB connection with srsUE?

### DISCUSSION THREAD

**Discussion by jboateng1** [srsran/srsRAN_Project#45] [General Help] [answered]
Hello srsRAN community,

I am working on an experimental testbed which has USRPs as gNBs and UEs. I am trying to test the new srsRAN 5G TDD code and I am wondering if anyone has tested the gNB with a USRP based UE (for instance B210)? I understand the srsUE from the previous code release supports only 15kHz scs.

Has anyone tried this setup(srsgNB and srsUE) using TDD bands or anyone has any idea how this may or may not work?

**Comment by gustavobsch** [comment, marked as answer]
Please see #32

**Reply by jboateng1** [reply]
Thanks
